Sources/TuistGenerator/Mappers/TargetActionEnvironmentMapper.swift

```diff
@@ -0,0 +1,20 @@
+import TuistCore
+
+public class TargetActionEnvironmentMapper: TargetMapping {
```

Comment:
Nitpick: I'd mark this `final`
I'd also add just a quick documentation that explains what this mapper is for

Reply:
Done